scanner_cli.py

```diff
@@ -205,21 +260,22 @@ def store_user_data(username):
         # Check if the username is already in the database
         if username in user_data:
             print(f"Hello, {username}!")
-            # print out current user profile
+            # Print out current user profile
             username_data = user_data[username]
             print("Here is the current user profile")
             print(f"Real Estate Minimum Price: ${username_data['min_price_range']}")
             print(f"Real Estate Maximum Price: ${username_data['max_price_range']}")
             print(f"District Rating Preference: {username_data['school_rating_preference']}/10")
             print(f"Gross Monthly Income: ${username_data['gross_income']}")
             print(f"Monthly Debt Payments: ${username_data['current_debt']}")
-            print(f"Management fees as percent of rental income: {username_data['current_debt']}%")
+            print(f"Management fees as percent of rental income: {username_data['management_fees']}%")
             print(f"Maintenance as percent of rental income: {username_data['maintenance_costs']}%")
             print(f"Insurance costs as percent of rental income: {username_data['insurance']}%")
-            # ask if we want to update
+            # Ask if we want to update
             update = input("Do you want to update the user profile ? (Y/N) ")
         else:
             print(f"New user {username}")
+            update = "Y"
```

Comment:
what is the update doing? why Y?

Reply:
update variable takes in user input after displaying current profile filters so we have the option to reinput fields